README.md

```diff
@@ -18,7 +18,7 @@ docker buildx build --platform linux/amd64 -t chemdash:latest .
 
 To start chemdash from the CLI:
 ```
-docker run --rm --name chemdash -p 8000:8000 -v /tmp:/tmp -v /var/tmp:/var/tmp -d chemdash:latest
+docker run --rm --name chemdash -p 8000:8000 -v /tmp:/tmp -v /var/tmp:/var/tmp -d chemdash:latest dataset_800.csv  --port 8000 --debug 
```

Comment:
The new CLI call w/ docker.  Follow w/ `docker logs -f chemdash` to see the logs.